recipe/build-locally.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
```

Comment:
It looks like these files crept in after running conda-smithy in the recipe directory. Could we remove them? Should only have the `meta.yaml` file in there.